src/Microsoft.AspNet.Mvc.Core/ViewComponents/JsonViewComponentResult.cs

```diff
@@ -16,9 +17,20 @@ public class JsonViewComponentResult : IViewComponentResult
         /// Initializes a new <see cref="JsonViewComponentResult"/>.
         /// </summary>
         /// <param name="value">The value to format as JSON text.</param>
-        public JsonViewComponentResult(object value)
+        public JsonViewComponentResult(object value) :
+            this(value, formatter: null)
```

Comment:
Bizarre line break.  Move colon down with `this` as in `: this(...)`.  Same for next constructor.